src/org/usfirst/frc/team467/robot/Logging.java

```diff
@@ -56,8 +59,17 @@ private static void setupDefaultLogging() {
 			System.out.println("Failed to create log file appender: " + e.getMessage());
 		}
 
+		try {
+			DriverStation station = DriverStation.getInstance();
+			String event = station.getEventName();
+			String type = station.getMatchType().name();
+			String match = Integer.toString(station.getMatchNumber());
```

Comment:
I'm not sure what the log match data is for but if it's for what I think it is, I think that there should be an if statement to check if any of the strings are null, if they are they should be reassigned as "No event data found" or "No match data found" instead of catching the null pointer by the string.join. I think this will be able to eliminate the need for a try-catch and it will be able to tell what specifically went wrong, further it will tell us other match data instead of just giving a blanket "no match data".